static/js/play_finish.js

```diff
@@ -52,7 +58,9 @@ let app = new Vue({
         playTime: 0,
         loggedIn: false,
 
-        played: PLAYED
+        played: PLAYED,
+
+        isMounted: false
```

Comment:
For this, there's a handy directive called `v-cloak` that hides the template until vue is mounted.